2-classes-db/task.md

```diff
@@ -21,31 +21,60 @@ To submit this homework write the correct commands after each question.
 
 ### 1. Show the date, transaction_no, description and amount for those transactions whose amount is between £30,000 and £31,000.
 ```sql
+SELECT date, transaction_no, description, amount FROM spends WHERE amount BETWEEN 30000 and 31000;
 
 ```
 ### 2. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'fee'.
 ```sql
+SELECT date, transaction_no, supplier_inv_no, description, amount FROM spends WHERE description LIKE '%fee%' ;
 
 ```
 ### 3. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'Fee'.
 ```sql
+SELECT date, transaction_no, supplier_inv_no, description, amount FROM spends WHERE description LIKE '%Fee%' ;
 
 ```
 ### 4. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'fee' (case insensitive). You will need to search 'https://www.postgresql.org/docs/' to solve this.
 ```sql
+SELECT date, transaction_no, supplier_inv_no, description, amount FROM spends WHERE LOWER(description) LIKE '%Fee%' ;
 
 ```
 ### 5. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose amount is £25,000, £30,000, £35,000 or £40,000.
 ```sql
+SELECT date, transaction_no, supplier_inv_no, description, amount from spends WHERE amount = 25000 OR amount = 30000 OR amount = 35000 OR amount = 40000;
```

Comment:
This works fine but can be difficult to maintain all of those "ORs" - can you find a way to match several different values in a column without a lot of ORs?